.github/workflows/cross_rs4s.yaml

```diff
@@ -6,20 +6,25 @@
 
 name: Cross RS4S
 on:
+  workflow_dispatch:  # should enable manual start of test
   pull_request:
     types: [ opened, reopened, synchronize, labeled ]
   schedule:
     # schedule runs only on the default branch. time is in UTC.
     # * is a special character in YAML so you have to quote this string.
     # run every night at 10:00PM UTC.
     - cron:  '0 22 * * *'
+  push:
+    branches:
```

Comment:
We don't want to run tests on equipment on merges to branches. Instead we run when the tag 'testhub' is set on a PR, and on schedule.